app/scripts/directives/overview/mobileClientRow.js

```diff
@@ -29,12 +29,27 @@
     var serviceBindingsVersion = APIService.getPreferredVersion('servicebindings');
     var isServiceInstanceReady = $filter('isServiceInstanceReady');
     var isMobileService = $filter('isMobileService');
+    var boundServices = 'Bound Services';
```

Comment:
`const BOUND_SERVICES = 'Bound Services';` ?

Reply:
They don't seem to use constants anywhere else in the codebase so for now we haven't added this